assets/js/components/HostsList.jsx

```diff
@@ -120,6 +124,7 @@ const HostsList = () => {
         title: 'Tags',
         key: 'tags',
         className: 'w-80',
+        filterFromParams: true,
```

Comment:
Are we enabling tags to use the searchparams filtering for something in particular? I'd either enable them all or enable only what we immediately need.

Reply:
It's enabled for feature purpose, I enabled it because I want to have tests for multiple filters. I think we set to all the table filters in an upcoming pr